portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,20 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Ana Mata's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Ana Mata's Portfolio</h1>
+      <p>I'm a junior at The University of Texas at El Paso majoring in Computer science.
+          My interests include web development and databases. 
+      </p>
+      <img src="Images/ana.jpg" style="height:50vh" alt="Ana Mata">
+      <p>Click here to get a random anime recomendation:</p>
       <button onclick="addRandomGreeting()">Hello</button>
```

Comment:
It looks like what this button does is give anime recommendations? It might be better to rename the button text to something better like 'Click here for anime recommendations!' so that viewers know what they're getting. Same thing with updating the function name.

Reply:
I have decided to move this code to the other PR. will fix on that branch!